### PR TITLE
moved deleted functions to public for better compiler errors.

### PR DESCRIPTION
**Pull request by kohei306**

### CODE REVIEW

**Comment by kohei306** [comment]
> @googlebot I signed it!

@googlebot I signed it!

**Comment by vslashg** [comment]
The "iff" is not a typo, it stands for "if and only if".

The rest of this change looks good to me; do you want to revert the comment change, and I'll pull this in?

**Comment by kohei306** [comment]
Hi @vslashg , I see, I didn't know the abbreviation. I removed the "fixed typo" part.

File: absl/base/internal/low_level_scheduling.h [hunk 1]
```diff
@@ -61,6 +61,8 @@ class SchedulingGuard {
  public:
   // Returns true iff the calling thread may be cooperatively rescheduled.
   static bool ReschedulingIsAllowed();
+  SchedulingGuard(const SchedulingGuard&) = delete;
+  SchedulingGuard& operator=(const SchedulingGuard&) = delete;
```
Comment (chabiraabdelaziz) [inline absl/base/internal/low_level_scheduling.h:65]: www.google.com

**Review by chabiraabdelaziz — APPROVED** [review]

**Pull request merged**